### DISCUSSION TITLE
压缩管理器无法正常使用

### DISCUSSION THREAD

**Discussion by Linux-qitong** [linuxdeepin/developer-center#4290] [Troubleshooting | 故障排查]
https://bbs.deepin.org/phone/zh/post/256215
按照longlong研发的要求，所有问题，先发到这里。
deepin-compressor 5.12.14-1 amd64

这个版本安装了没法使用，打不开，右键也无反应

应用商店很多人在反馈，一个多月了官方都没有改！！！

谁有旧版的发个用用下，谢谢

装了360压缩跟7-zip都无法在右键里面显示

**Comment by BLumia** [comment]
根据原帖下面测试人员未能复现相关问题。提供点详细信息？

比如什么环境下，v20.9 还是 v23？

打不开是完全没界面？从终端启动的话有异常输出吗？如果有的话，输出内容是什么？

**Reply by Linux-qitong** [reply]
已经转发研发回复。
